### DISCUSSION TITLE
Anyone sucessful with maxChargeCurrentAC_A and e-Up?

### DISCUSSION THREAD

**Discussion by hmei67** [tillsteinbach/WeConnect-mqtt#366] [Q&A]
Hi all, I'm using WeConnect Mqtt in Fhem for quite a while now with a VW e-Up. Thanks to Till for the fine work.

Earlier I was able to set the maxChargeCurrentAC from "maximum" to "reduced" and vice versa without a problem. The charging power was then set to 6A or 16A. 

Since the newer maxChargeCurrentAC_A is also available, setting does not work anymore, but reading works Ok. Looking at the VW app, this setting could possibly be set to 5/10/13/32A. When I set it in the App, the reading of the Mqtt device in fhem is updated to this value. But setting it in fhem does not work for me. 

These are the writeable topics I'm using:
weconnect/vehicles/.../domains/charging/chargingSettings/maxChargeCurrentAC_writetopic
weconnect/vehicles/.../domains/charging/chargingSettings/maxChargeCurrentAC_A_writetopic

Has anyone been succesfull with e-Up and maxChargeCurrentAC_A setting the charging current?

  Hermann

**Comment by tillsteinbach** [comment]
Do you see any errors after setting the topic?

**Reply by hmei67** [reply]
This is "info":

Upstream service responded with an unexpected status. If the problem persists, please contact our support.

**Reply by tillsteinbach** [reply]
And when you change the setting through the app, does it change on Mqtt too?

**Reply by tillsteinbach** [reply]
And when you change the setting through the app, does it change on Mqtt too?

**Reply by hmei67** [reply]
Yes. It shows 5, 10 etc. what I entered in the App.

**Reply by bnavigator** [reply]
Same here
